pkg/ride/functions_proto.go

```diff
@@ -322,6 +339,23 @@ func bytesFromState(env Environment, args ...rideType) (rideType, error) {
 	return rideBytes(entry.Value), nil
 }
 
+func bytesFromSelfState(env Environment, args ...rideType) (rideType, error) {
+	k, err := extractKey(args)
+	if err != nil {
+		return rideUnit{}, nil
```

Comment:
why did you decided to not return error we could get?

Reply:
Because this functions work like this in Scala node.